packages/dcos-integration-test/extra/test_composition.py

```diff
@@ -87,9 +87,9 @@ def test_signal_service(dcos_api_session):
     if enabled == 'false':
         pytest.skip('Telemetry disabled in /opt/mesosphere/etc/dcos-signal-config.json... skipping test')
 
-    logging.info("Version: ", dcos_version)
```

Comment:
I personally hate %-based format strings. You can make the argument that .format and f-strings are just as bad, but no one will ever have to lookup %i vs %d vs %s